components/ActionSheet/src/MDCActionSheetController.m

```diff
@@ -164,6 +164,11 @@ - (void)addAction:(MDCActionSheetAction *)action {
   return [_actions copy];
 }
 
+- (void)loadView {
+  [super loadView];
+  self.mdc_bottomSheetPresentationController.delegate = self;
```

Comment:
Nit: New line after super to follow conventions of the file.

Reply:
Done